bindata/manifests/switchdev-config/files/switchdev-configuration-after-nm.sh.yaml

```diff
@@ -25,11 +43,25 @@ contents:
         echo "load VF driver for $pci_addr"
         VfDirs=$(ls /sys/bus/pci/devices/${pci_addr} | grep virtfn)
 
-        # load VF driver
+        # load VF driver and configure vdpa if needed
         for VfDir in $VfDirs
         do
           VfPciAddr=$(basename "$( readlink -f /sys/bus/pci/devices/${pci_addr}/$VfDir )")
           echo $VfPciAddr > /sys/bus/pci/drivers_probe
+
+          # extract VF id from VfDir
+          vfid=${VfDir:6}
```

Comment:
why 6 ?

Reply:
it extracts the VF id from strings like `/sys/bus/pci/devices/0000:65:00.0/virtfn1`. 1 is the 7th character after `virtfn`. Is it enough to add a comment above the line in question?

Reply:
a comment would be nice :) thx !